src/main/java/com/schematica/world/storage/Schematic.java

```diff
@@ -21,6 +21,8 @@
 public class Schematic implements ISchematic
 {
     private static final ItemStack                              DEFAULT_ICON   = new ItemStack(Blocks.GRASS);
+    //todo: check if there is a non-internal way to use the registry
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)